internal/resource/config/pingoneconnection/ping_one_connection_resource.go

```diff
@@ -0,0 +1,295 @@
+package pingoneconnection
```

Comment:
Is this file supposed to be here?

Reply:
oops, left from another branch in the works, good catch.